lib/ember-cli/asset_resolver.rb

```diff
@@ -1,8 +1,8 @@
 module EmberCLI
   class AssetResolver
-    def initialize(options)
-      @app = options.fetch(:app)
-      @sprockets = options.fetch(:sprockets)
+    def initialize(app:, sprockets:)
```

Comment:
They're [required keyword arguments](https://robots.thoughtbot.com/ruby-2-keyword-arguments#required-keyword-arguments).

[This PR](https://github.com/thoughtbot/ember-cli-rails/pull/227) is also related.